### ISSUE TITLE
New design challenge

### ISSUE THREAD

**Issue by paskal** [kAworu/octostrap3#50]
Hello, Alexandre!

I was not sure how to contact you so creating an issue here. Octopress seems like a sunken boat and it's time to move on. The hot thing now (and I'm not sure Octopress ever had such popularity) is Hugo.
I've tried to migrate my blog to Hugo and was surprised by lack of good themes for blogs. "[Beg](https://github.com/dim0627/hugo_theme_beg)" is one similar to this one but it have huge room for improvement which I'm not able fill not being a designer myself.
Long story short, would you be so kind to apply your genius to make similar theme look a little closer to perfection which you've achieved here? Here is the issue: https://github.com/dim0627/hugo_theme_beg/issues/22

P.S. I miss our collaboration very much, it's sad we can't tweak this thing anymore:)

**Comment by kaworu** [comment]
Hi @paskal and happy new year :)

Agreed that it's time to move on, I've switched to [nanoc](https://nanoc.ws/) a while ago.

I'm super happy of nanoc and you should give it a try. It's ultra flexible, still Ruby (made my switch less painful) and - more importantly - all the frustrations I had with Octopress are gone. On the other hand it demands to be more involved as you basically configure everything and themes are not as reusable as they were with Octopress.

I strongly believe that the best projects are the one where its developers use it themselves (i.e. "eat your own dog food"). Consequently, I don't think it would be reasonable to start and maintain a Hugo theme myself. The "Beg" theme looks really similar to what we've developed with Octostrap3, the diff looks small and mainly CSS (e.g. article titles, dates etc.).

Many thanks for the praise though, it was a blast to collaborate with you on this project and I look forward the next time we get the chance to work together! In the mean time I wish you all the best.

Alex.
